core/camel-api/src/main/java/org/apache/camel/ResumeStrategy.java

```diff
@@ -21,12 +21,19 @@
  * Defines a strategy for handling resume operations. Implementations can define different ways to handle how to resume
  * processing records.
  */
-public interface ResumeStrategy<T> {
+public interface ResumeStrategy {
 
     /**
      * A consumer, iterator or value class that can be used to set the index position from which to resume from. The
      * type is specific to the component.
      *
      */
     void resume();
+
+    /**
+     * Starts the resume strategy
+     * 
+     * @throws Exception
+     */
+    void start() throws Exception;
```

Comment:
Its IMHO better to extend Service that comes with lifecycle apis start/stop etc and use that, instead of your own "start" apis